packages/react-dom/src/server/DOMMarkupOperations.js

```diff
@@ -80,30 +68,29 @@ export function createMarkupForRoot() {
  * @return {?string} Markup string, or null if the property was invalid.
  */
 export function createMarkupForProperty(name, value) {
+  if (name !== 'style' && shouldSkipAttribute(name)) {
+    return '';
+  }
+  if (shouldTreatAttributeValueAsNull(name, value)) {
+    value = null;
+  }
+  if (value === null) {
+    return '';
+  }
```

Comment:
Could you just `return ''` inside of the `shouldTreatAttributeValueAsNull` block? When does `shouldTreatAttributeValueAsNull` return false but the value is null?

Reply:
Nice catch